pkg/antctl/portforwarder/portforwarder.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2022 Antrea Authors
```

Comment:
I am curious as to why we are placing this code in `pkg/antctl` given that at the moment at least, it is only used for e2e testing?

Reply:
Yes, then should I create new folder for it ?

Reply:
yes, I think `test/e2e/utils/portforwarder` would make sense